lua/gitlinker/hosts.lua

```diff
@@ -171,7 +171,7 @@ end
 function M.get_matching_callback(target_host)
   local matching_callback
   for host, callback in pairs(M.callbacks) do
-    if host:match(target_host) then
+    if target_host:match(host) then
```

Comment:
nice catch, makes much more sense now.

now it's even possible to use lua regexes in the `hosts` configuration table in `setup()`

Reply:
Yeah that's the point